src/components/Atoms/Alert/index.jsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+
+import styled from "styled-components";
+
+import COLOR from "../../../variables/color";
+import TEXT from "../../../variables/texts";
+import BREAKPOINT from "../../../variables/breakpoint";
+import FONT_FAMILY from "../../../variables/font_family";
+
+export const Alert = ({ isActive, errorText = "" }) => {
+  return <StyledText isActive={isActive}>{errorText}</StyledText>;
+};
+
+const StyledText = styled.div`
+  position: absolute;
+  top: 80px;
+  left: 50%;
+  transform: translateX(-50%);
```

Comment:
```suggestion
  transform: translateX(-50%) translateY(-40px);
```
translateY(-40px) がないと、フェードイン・アウトが出来ません。